package.json

```diff
@@ -56,7 +56,7 @@
   "binary": {
     "module_name": "bcrypt_lib",
     "module_path": "./lib/binding/",
-    "host": "https://github.com",
-    "remote_path": "/kelektiv/node.bcrypt.js/releases/download/v{version}/"
+    "host": "https://github.com/kelektiv/node.bcrypt.js/releases/download/",
+    "remote_path": "v{version}"
```

Comment:
I found an artifacts section inside AppVeyor. And Travis documentation only gives about info about automatic github release uploading